docs/operation_center/faq.md

```diff
@@ -0,0 +1,31 @@
+# Operation Center: Frequently Asked Questions
+
+## Is the IP behind `intake.sekoia.io` static?
+
+**IP for `intake.sekoia.io` is `145.239.192.38`.**
+
+`intake.sekoia.io`, used to send your logs to SEKOIA.IO, either via Syslog or HTTP protocols. IP behind that service is static and stable. You can use that IP to configure your firewalls to only allow some of your device to forward their logs to SEKOIA.IO.
+
+## How to debug Rsyslog’s forward configuration to SEKOIA.IO?
+
+If you use Rsyslog to forward you logs to SEKOIA.IO, you will probably have a section like this in your configuration files:
+
+```
+template(name="SEKOIAIOUnboundTemplate" type="string" string="<%pri%>1 %timestamp:::date-rfc3339% %hostname% %app-name% %procid% LOG [SEKOIA@53288 intake_key=\"jOK5bMVXz5Iz7gfogQDbCcC7l7S2IrOs5\"] %msg%\n")
```

Comment:
is it ok to have the intake key here ?

Reply:
Yes, this is a random key generated for that documentation.